sound/dynamic_mixer.h

```diff
@@ -185,7 +185,10 @@ template<int N> class AudioDynamicMixer : public ProffieOSAudioStream, Looper {
   int32_t peak_sum_ = 0;
   int32_t peak_ = 0;
   int32_t num_samples_ = 0;
-  int32_t volume_ = VOLUME;
+#ifndef BOOT_VOLUME
```

Comment:
The #ifdef should go in ProffieOS.ino so that BOOT_VOLUME can be used anywhere.

Reply:
ah. Ok done.